postman/national_record_locator_producer_sandbox.json

```diff
@@ -0,0 +1,398 @@
+{
+	"info": {
+		"_postman_id": "b503cff1-8a65-48c0-8479-58900a1c6350",
+		"name": "National Record Locator Producer Sandbox",
+		"description": "Sandbox example requests",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "5426004"
+	},
+	"item": [
+		{
+			"name": "Create Pointer",
+			"protocolProfileBehavior": {
+				"disabledSystemHeaders": {
+					"accept": true
+				}
+			},
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "nhsd-correlation-id",
+						"value": "nrl-consumer-api-sandbox",
+						"type": "text"
+					},
+					{
+						"key": "x-correlation-id",
+						"value": "nrl-consumer-api-sandbox",
+						"type": "text"
+					},
+					{
+						"key": "NHSD-End-User-Organisation-ODS",
+						"value": "RY26A",
+						"type": "text"
+					},
+					{
+						"key": "Authorization",
+						"value": "auth",
+						"type": "text"
+					},
+					{
+						"key": "x-request-id",
+						"value": "create-pointer-test",
+						"type": "text"
+					},
+					{
+						"key": "accept",
+						"value": "version=1.0",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
```

Comment:
change the mode to 'json' where a request body is being passed through the file

Reply:
"language": "json" added.